### DISCUSSION TITLE
Filter field with condition

### DISCUSSION THREAD

**Discussion by fariasfc** [carltongibson/django-filter#1429] [Q&A]
Is it possible to filter a field using conditions?

Ex:
If I have a DateField/DateTimeField, I would like to filter like this:

>15/05/2020  would be equivalent to field__gt=15/05/2020
<=15/05/2020  would be equivalent to field__lte=15/05/2020

**Comment by king-11** [comment]
@felipefariax you can specify the default lookup_expr and use two different fields because I believe it isn't possible to check for <= operator while filtering
